tests/baselines/reference/privateNamesAndkeyof.types

```diff
@@ -16,5 +16,5 @@ class A {
 }
 
 type T = keyof A     // should not include '#foo'
->T : "bar" | "baz"
+>T : keyof A

```

Comment:
This now fails to prove the test case as it does not provide enough information.